spec/features/conference_registration_spec.rb

```diff
@@ -49,5 +65,16 @@
         expect(conference.user_registered?(participant)).to be(true)
       end
     end
+
+    context 'registration is closed' do
+      let(:conference_with_closed_registration) { create(:conference, registration_period: create(:registration_period)) }
+
+      scenario 'registers for a conference', feature: true do
+        participant.is_admin = false
```

Comment:
In the beginning you can do 
let!(:first_user) { create(:user) }

So that the rest of the code does not make us wonder why.